scripts/helmAcs.sh

```diff
@@ -254,6 +271,9 @@ messageBroker:
 share:
   livenessProbe:
     initialDelaySeconds: 420
+  image:
+    repository: \"$SHARE_IMAGE\"
+    tag: \"$SHARE_TAG\"
```

Comment:
A similar if condition here too for tag and repository. We only need to override if != "" 
Otherwise we can use the chart values